### PR TITLE
Update gt7-extramenus.py

### PR DESCRIPTION
**Pull request by dehydratedH2O**
improved runtime by roughly 20sec per cycle by skipping extra waits and inputs for engine ticket acceptance. tested overnight on PS5 using GT7 physical disc.

### CODE REVIEW

File: gt7-extramenus.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def start_exploit(section):
     press("escape")
     press("escape")
     press("escape")
-    sleep(4)
+    sleep(2.5)
```
Comment (mistakenchicken) [inline gt7-extramenus.py:105]: Please discard this change, in my testing, sometimes it could take a bit longer to load the menu.

**Review by mistakenchicken — CHANGES_REQUESTED** [review]
Hi, thank you so much for your contribution! I will merge as soon as you address the comment on the pull request. The rest looks good!

**Comment by dehydratedH2O** [comment]
reverted to original timing as requested for that one line

**Comment by mistakenchicken** [comment]
Thank you @dehydratedH2O and sorry for taking so long to merge, I was quite busy this weekend.

**Pull request merged**